DEPRECATION.md

```diff
@@ -44,10 +44,12 @@ features:
 | Deprecated | [Absolute import for `@cubejs-backend/server-core`](#absolute-import-for-@cubejs-backendserver-core)               | v0.25.4    | v0.30.0   |
 | Deprecated | [Absolute import for `@cubejs-backend/schema-compiler`](#absolute-import-for-@cubejs-backendschema-compiler)       | v0.25.21   | v0.32.0   |
 | Deprecated | [`checkAuthMiddleware`](#checkauthmiddleware)                                                                      | v0.26.0    |           |
-| Deprecated | [Node.js 10](#nodejs-10)                                                                                           | v0.26.0    |           |
+| Deprecated | [Node.js 10](#nodejs-10)                                                                                           | v0.26.0    | v0.29.0   |
+| Deprecated | [Node.js 15](#nodejs-15)                                                                                           | v0.26.0    | v0.29.0   |
```

Comment:
Shouldn't this be `Removed` instead of `Deprecated`?

Reply:
Yes, let's use Removed. Fixed, Thanks!